lib/async_hooks.js

```diff
@@ -60,19 +60,11 @@ async_wrap.setupHooks({ init,
                         destroy: emitDestroyN });
 
 // Used to fatally abort the process if a callback throws.
-function fatalError(e) {
```

Comment:
> By "exporting `FatalException`" do you mean expose it to JS?

Yes. At the very least I would like a unified way of making a fatal exception. Currently, there are a couple of different implementations floating around in nodecore. It makes things like `--abort-on-uncaught-exception` difficult to test.